src/main/java/com/spring/app/controllers/CompanyController.java

```diff
@@ -0,0 +1,42 @@
+package com.spring.app.controllers;
+
+import com.spring.app.domain.Company;
+import com.spring.app.dto.CompanyDTO;
+import com.spring.app.services.CompanyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * CompanyController
+ * All activities with companies
+ *
+ * @author lyubov
+ */
+@RestController
+@RequestMapping("/companies")
+public class CompanyController {
+
+    private CompanyService service;
+
+    public CompanyController(@Autowired CompanyService service){
+        this.service = service;
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Provide Media type on the top of the controller

Reply:
I tried adding options in @RequestController (produces and consumes together) but I got a conflict with spring inner classes, then I found the @ResponseBody, which helps me archive my purpose

Reply:
@LetiLubov wat?